rsocket-core/src/main/java/io/rsocket/frame/FrameUtil.java

```diff
@@ -5,6 +5,8 @@
 import io.netty.buffer.Unpooled;
 
 public class FrameUtil {
+  public static final int FRAME_MAX_SIZE = 16_777_215;
```

Comment:
@robertroeser I did not find a place which limits `mtu` to that size? Is that limitation still valid?

Reply:
https://github.com/rsocket/rsocket-java/blob/develop/rsocket-core/src/main/java/io/rsocket/frame/FrameLengthFlyweight.java#L17

Reply:
Can you switch the length to  0xFFFFFF?

Reply:
Done. Uses `FRAME_LENGTH_MASK` from `FrameLengthFlyweight`